### PR TITLE
Request location

### PR DESCRIPTION
**Pull request by Amele-A**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,30 +1,43 @@
...
+          <Stack.Screen
+            name="Search"
+            component={Search}
+          />
+
+          <Stack.Screen
+            name="Main"
+            component={Aqi}
+          />
+        </Stack.Navigator>
```
Comment (kmausolf) [inline App.tsx:36]: Very clean, I love how easy it is to set up a nav stack with `react-navigation`!

File: screens/aqi_screen/aqiPage.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
+import React, { FC } from "react";
+import { Text, View, StyleSheet, ImageBackground, Pressable } from "react-native";
+
+import { Ionicons, MaterialIcons } from "@expo/vector-icons";
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { StackParamList } from '../../App';
+
+type AQIScreenProps = NativeStackScreenProps<StackParamList, "Main">;
```
Comment (kmausolf) [inline screens/aqi_screen/aqiPage.tsx:8]: Nice job inferring the type here.  I'm glad they got around to adding TS bindings to `react-navigation`.

File: screens/search_screen/search.tsx [hunk 3]
```diff
@@ -1,64 +1,85 @@
...
 
-interface Props {
-  city: string;
-  setCity: React.Dispatch<React.SetStateAction<string>>;
-}
-
-const Search: FC<Props> = (props) => {
+      let currentLocation: any = await Location.getCurrentPositionAsync({});
+      setLocation(currentLocation);
+    })();
```
Comment (kmausolf) [inline screens/search_screen/search.tsx:32]: Nicely done with the cleanup function.  It took me a lot of head scratching to figure out how to do an async call from within `useEffect`

File: screens/search_screen/search.tsx [hunk 4]
```diff
@@ -1,64 +1,85 @@
...
+  }, []);
   return (
     <View style={styles.container}>
-      <Pressable style={styles.locationIcon} >
+      {/* <Pressable style={styles.locationIcon}  >
         <Ionicons name="ios-location-outline" size={34} color="black" />
-      </Pressable>
+      </Pressable> */}
```
Comment (kmausolf) [inline screens/search_screen/search.tsx]: Is this still needed?

**Review by kmausolf — APPROVED** [review]
Great work here, Amele.  In the future, be sure to include a PR description!  It's helpful for understanding the context of the work you've done.

File: screens/search_screen/search.tsx (hunk 4 above)
Comment (lindakw) [reply screens/search_screen/search.tsx]: This is no longer needed and will be removed.

**Review by lindakw — CHANGES_REQUESTED** [review]
The useEffect looks good! We do need to remove the location icon from the search page.

**Review by lindakw — APPROVED** [review]

**Pull request merged**